Odata-docs/webapi-8/fundamentals/overview.md

```diff
@@ -0,0 +1,581 @@
+---
+title: "ASP.NET Core OData 8 Fundamentals Overview"
+description: "This document introduces the fundamental concepts of ASP.NET Core OData library."
+
+author: habbes
+ms.author: clhabins
+
+ms.date: 1/5/2023
+---
+# Fundamentals Overview
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+This article provides an overview of the fundamental concepts for building ASP.NET Core OData services.
+
+ASP.NET Core OData is a .NET library that sits on top of ASP.NET Core to help you build OData REST APIs. The ASP.NET Core OData library consists of a set of services and middleware that hook into ASP.NET Core's request pipeline to provide features like routing, query handling and serialization based on OData specifications and conventions. All these concepts will be covered in depth in separate articles, this just provides an overview.
+
+To build an ASP.NET Core OData app, you start with an ASP.NET Core application, then install the [`Microsoft.AspNetCore.OData`](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) package as a dependency from NuGet. ASP.NET Core OData 8 supports .NET Core 3.1 and .NET Core 6.0 and above.
+
+> [!NOTE]
+> ASP.NET COre OData 8 supports .NET Core 3.1 and .NET Core 6.0 and above.
+
+# [Visual Studio](#tab/visual-studio)
+
+In the Visual Studio **Package Manager Console**:
+
+```powershell
+Install-Package Microsoft.AspNetCore.OData
+```
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+```dotnetcli
+dotnet add package Microsoft.AspNetCore.OData
+```
+
+---
+
+Internally, ASP.NET Core OData depends on the following.NET libraries for OData: 
+- [`Microsoft.OData.Core`](https://www.nuget.org/packages/Microsoft.OData.Core), which provides features like reading and writing OData payloads, parsing OData URIs and query options
+- [`Microsoft.OData.Edm`](https://www.nuget.org/packages/Microsoft.OData.Edm), which provides support for working with OData schemas, also referred to as EDM (Entity Data Model) model.
+
```

Comment:
how come you didn't list the spatial library here.

Reply:
I didn't think it was significant enough to be mentioned. But I can add it.